.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.3.1
+2.3.1
```

Comment:
I'm not sure if this is something that should be pushed; it was done for me to get rbenv working locally but I think we use rvm on the server so this should probably remain as it is.

Reply:
If you're not sure and you haven't tested it then revert it.